examples/claude-try-it/install.ps1

```
@@ -779,6 +1254,111 @@ function Write-Summary {
     Write-Host ""
 }
 
+# --- Stop stale MCP server processes --------------------------------------
+
+function Stop-StaleProcesses {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Stop-StaleProcesses' uses a plural noun. A singular noun should be used instead.

Suggested fix: Singularized correction of 'Stop-StaleProcesses'

(PSUseSingularNouns)